src/aruco_pose/config/fiducial_marker_param.yaml

```diff
@@ -18,6 +18,9 @@ aruco_pose:
     offset_on_marker_x: -0.005
     offset_on_marker_y: 0.0
     fiducial_marker_family: 'DICT_APRILTAG_36h11'
-    physical_marker_size: 0.02665 #
+    physical_marker_size: 0.02665 # measure the printed tag size
     image_topic: "/rgb/image_raw"
     number_of_observations: 10 # size of the moving window for the median filter
+    cam_to_lense_x: -0.032
+    cam_to_lense_y: 0.0
+    cam_to_lense_z: 0.0
```

Comment:
Lens is spelt 'lens'

Reply:
Correct spelling is added:
https://github.com/maffettone/erobs/blob/01dec49d475b66d9f6bcae9cbe1f5dcb73c92be9/src/aruco_pose/config/fiducial_marker_param.yaml#L16-L18